conf/modules/telemetry_transparent_secure_example.xml

```diff
@@ -3,20 +3,26 @@
 <module name="telemetry_transparent_secure" dir="datalink" task="datalink">
```

Comment:
I think the module (and file) names should reflect the type of encryption in case we want to implement another one